src/java-runtime/java/mono/android/Runtime.java

```diff
@@ -14,12 +14,13 @@ public class Runtime {
 		Thread.setDefaultUncaughtExceptionHandler (new XamarinUncaughtExceptionHandler (Thread.getDefaultUncaughtExceptionHandler ()));
 	}
 
-	public static native void init (String lang, String[] runtimeApks, String runtimeDataDir, String[] appDirs, ClassLoader loader, String[] externalStorageDirs, String[] assemblies, String packageName, int apiLevel, String[] environmentVariables);
+	public static native void init (String lang, String[] runtimeApks, String runtimeDataDir, String[] appDirs, int localDateTimeOffset, ClassLoader loader, String[] externalStorageDirs, String[] assemblies, String packageName, int apiLevel, String[] environmentVariables);
```

Comment:
I think adding a parameter to `init` here is breaking the Designer tests -- they call this API directly.

Can we just pass a 0 or some default value through? I don't think it matters if the behavior for `DateTime.Now` is off a little bit when rendering layouts in the designer.

Reply:
@jonathanpeppers Ah, I didnt realize that was the root of the failures on the unit tests. I'm not actually sure what the corresponding designer tests are, but do any actually leverage `DateTimeOffset.Now`?

Whats the easiest way to confirm that adding a 0 or default parameter on the designer's `init` call will fix the tests? If it does solve the problem, then would the correct order be to merge this pr knowing that it will break tests and then merge a pr on designer to add the parameter?

Or would it be better to make another `init` like having both
```
public static native void init (String lang, String[] runtimeApks, String runtimeDataDir, String[] appDirs, ClassLoader loader, String[] externalStorageDirs, String[] assemblies, String packageName, int apiLevel, String[] environmentVariables);
public static native void init (String lang, String[] runtimeApks, String runtimeDataDir, String[] appDirs, int localDateTimeOffset, ClassLoader loader, String[] externalStorageDirs, String[] assemblies, String packageName, int apiLevel, String[] environmentVariables);
```

Reply:
I would guess that no designer tests use `DateTimeOffset.Now`. It doesn't seem like something you would do in a custom `UserControl`-like scenario.

But they do call this `init()` method with a known set of parameters -- we can't change that or the designer will crash on startup.